### ISSUE TITLE
image_histogram2d throw size mismatch exception with BxCxHxW input and return_pdf=True

### ISSUE THREAD

**Issue by lyhyl** [kornia/kornia#1294]
### Describe the bug

```
File ./kornia/enhance/histogram.py, line 255
Exception has occurred: RuntimeError
The size of tensor a (128) must match the size of tensor b (4) at non-singleton dimension 2
```

### Reproduction steps

```bash
1. get data from a batch data loader (128x4x32x32 in my case)
2. feed data into `image_histogram2d(x, min, max, bins, kernel=kernel, return_pdf=True)`
```

### Expected behavior

return correct pdf without exception

### Environment

```shell
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```
- PyTorch Version: 1.9.0
- OS: Windows 10 x64
- How you installed PyTorch: conda install -c conda-forge kornia
- Build command you used (if compiling from source): N/A
- Python version: 3.8
- CUDA/cuDNN version: -
- GPU models and configuration: 3080
- Any other relevant information: -

### Additional context

In file histogram.py, line 255
```
normalization = torch.sum(hist, dim=-1).unsqueeze(0) + eps
```
it sum `hist` at last dim (bins dim) but unsqueeze at dim 0, seems weird.
maybe it should be
```
normalization = torch.sum(hist, dim=-1, keepdim=True) + eps
```

**Comment by edgarriba** [comment]
@lyhyl thanks for reporting the issue. Could you go a PR fixing ?

**Comment by lyhyl** [comment]
I have made a PR https://github.com/kornia/kornia/pull/1295
